snuba/admin/clickhouse/querylog.py

```diff
@@ -31,6 +34,18 @@ def describe_querylog_schema() -> ClickhouseResult:
     return __run_querylog_query(f"DESCRIBE TABLE {schema.get_table_name()}")
 
 
+def _get_clickhouse_threads() -> int:
+    config_threads = state.get_config("admin.querylog_threads", _MAX_CH_THREADS)
+    try:
+        return min(
+            int(config_threads) if config_threads is not None else _MAX_CH_THREADS,
+            _MAX_CH_THREADS,
+        )
+    except ValueError:
+        # in case the config is set incorrectly
```

Comment:
I don't think we should swallow this error. Otherwise someone will think they successfully changed the threads.

Reply:
done